source/direct_debit/index.html.md.erb

```diff
@@ -5,248 +5,126 @@ weight: 80
 
-### Variable Direct Debit payment process
+- be longer than 255 characters
+- contain URLs
 
-These steps happen in order.
+The format is variable.
 
-#### 1. GOV.UK Pay creates the agreement
+#### return_url
 
-API call:  `POST /v1/agreements/`
+`return_url` is an HTTPS URL on your site. GOV.UK Pay will direct your user to this url when they've finished setting up the mandate.
 
-Your service must specify:
+The URL should not be JSON-encoded, because backslashes are invalid characters in the API.
 
-* a `return_url` where GOV.UK Pay will direct the user when they have finished setting up the agreement
 
-* the `agreement_type` (`on demand`)
+### 2. Direct the user to GOV.UK Pay
 
-* a `reference` (optional) - your reference for this agreement
+When the API responds, you need to:
 
-The response contains a `next_url`.
+- direct the user to the `next_url` from the API response
+- store the `mandate_id` from the API response, so you can use it to collect payments against the mandate later
 
-#### 2. Your service directs the user to the `next_url`
+Refer to the <a
+href="https://govukpay-api-browser.cloudapps.digital/#newpayment"
+target="blank">GOV.UK Pay API browser</a> or the <a href="https://github.com/alphagov/pay-publicapi/blob/master/swagger/swagger.json" target="blank">Swagger file</a> for more
+information on the API response.
 
-#### 3. The user provides their account details
+### 3. The user provides their bank account details
 
-For example, account number and sort code for Direct Debit.
+When your service redirects the user to `next_url`, they’ll see a **Set up a Direct Debit** page. The user enters their bank account details and selects **Continue**.
 
-GOV.UK Pay makes an API call to our Direct Debit supplier to verify account
+GOV.UK Pay makes an API call to GoCardless to verify the user's account
 details.
 
-#### 4. GOV.UK Pay presents the user with a confirmation screen
+### 4. GOV.UK Pay presents the user with a confirmation screen
 
 The screen presents the user’s account details. The user grants their consent
 by selecting __Confirm__.  GOV.UK Pay sends an email to the user to confirm
 that their mandate has been set up. If the mandate subsequently fails, the
 GOV.UK Pay team will send an email to update the user that their mandate could
 not be set up.
 
-#### 5. GOV.UK Pay securely passes the user's details on to the Direct Debit provider
+### 5. GOV.UK Pay securely passes the user's details on to the Direct Debit provider
 
 The Direct Debit provider submits a request to BACS to set up a mandate, and
 sends a mandate ID to GOV.UK Pay. At this point, services can send a payment
 request.
 
-#### 6. GOV.UK Pay returns the user to your service using the `return_url`
+### 6. GOV.UK Pay returns the user to your service using the `return_url`
+
+### 7. Call the API to check the status of the mandate
+
+API call: `GET /v1/directdebit/mandates/<mandateid>`
+
+This is to display an appropriate screen to the user.
```

Comment:
I don’t understand what this means.

Reply:
This phrasing was borrowed from the 'Payment flow' page, but I agree that it isn't very clear. I've updated in https://github.com/alphagov/pay-tech-docs/pull/298/commits/562c05d6cc099a67190a4bfe298a6acc6291794c with clearer language from elsewhere in 'Payment flow'.